update package.json and CHANGELOG for release 4.3.0 (#88)
blackbaud/skyux-ag-grid: CHANGELOG.md

# 4.3.0 (2020-12-02)

> - Added modern theme styling for ag-Grid. [#83](https://github.com/blackbaud/skyux-ag-grid/pull/83)

Reviewer: Should `ag-Grid` be `the data entry grid component` since that's what we call it on the docs site? Same question about the next entry. We have multiple instances of `ag-Grid` in the changelog, but we also have instances of `data entry grid`. I'm not when one is appropriate and when the other is.

Author: I'll use data entry grid for this one since its talking about the component, but the next entry is talking about the ag-grid libraries themselves so I'll leave those

> - Updated the ag-Grid peer dependencies to allow for any version. [#87](https://github.com/blackbaud/skyux-ag-grid/pull/87)

Reviewer 2: ```suggestion
- Updated the peer dependencies to support the latest version of `ag-grid-angular` and `ag-grid-community`. [#87](https://github.com/blackbaud/skyux-ag-grid/pull/87)
```

> - Updated the ag-Grid peer dependencies to allow for any version. [#90](https://github.com/blackbaud/skyux-ag-grid/pull/90)

Reviewer 2: Does this work?

```suggestion
- Updated the peer dependencies to support the latest version of `ag-grid-angular` and `ag-grid-community`. [#90](https://github.com/blackbaud/skyux-ag-grid/pull/90)
```
